docs/rules/no-mixed-operators.md

```diff
@@ -9,6 +9,21 @@ var foo = (a && b) || c || d;  /*GOOD*/
 var foo = a && (b || c || d);  /*GOOD*/
 ```
 
+**Note:**
+It is expected for this rule to emit one error for each mixed operator in a pair. As a result, for each two consecutive mixed operators used, a distinct error will be displayed, pointing to the where the specific operator that breaks the rule is used:
```

Comment:
> pointing to the where...

"the" should be removed?

Reply:
Oops! Thanks @Aladdin-ADD, I removed it.